samples/data-grid/basic/data-grid-2/demo.css

```diff
@@ -8,3 +8,48 @@
     margin-bottom: 15px;
     height: 400px;
 }
+
+.highcharts-dg-row[row-index="1"] {
+    background-color: #afa;
+    height: 100px;
+}
+
+.highcharts-dg-row[row-index="3"] {
+    background-color: #afa;
+    height: 100px;
+}
+
+.highcharts-dg-row[row-index="6"] {
+    background-color: #afa;
+    height: 100px;
+}
+
+.highcharts-dg-row[row-index="100"] {
+    background-color: #f0a;
+    height: 200px;
+}
+
+.highcharts-dg-row[row-index="150"] {
+    background-color: #aff;
+    height: 10px;
+}
+
+.highcharts-dg-row[row-index="152"] {
+    background-color: #aff;
+    height: 10px;
+}
+
+.highcharts-dg-row[row-index="154"] {
+    background-color: #aff;
+    height: 10px;
+}
```

Comment:
> No issues with the code at first sight but the grid looks strange when scrolling to row 145. Is it the intention to show only every second row fully?

@jomar-honsi Yes, this is the height set for these rows in CSS intentionally, to check how it behaves when their height is not only higher than the default height, but also lower.